railties/lib/rails/generators/rails/app/templates/Dockerfile.tt

```diff
@@ -0,0 +1,33 @@
+# Make sure it matches the Ruby version in .ruby-version and Gemfile
+FROM ruby:<%= Gem.ruby_version %>
+
+<% if using_node? -%>
+# Install JavaScript dependencies and libvips for Active Storage
+RUN curl -sL https://deb.nodesource.com/setup_19.x | bash -
+RUN apt-get update -qq && apt-get install -y build-essential libvips nodejs && npm install -g yarn
```

Comment:
Unless it's purely for dev, it's recommended to run `&& rm -rf /var/lib/apt/lists/*` after each apt-get update to avoid bloat in layers.

Reply:
It's not for dev, this is specifically for production, so let's add that 👍